js/src/product-attributes/index.js

```diff
@@ -1,4 +1,45 @@
+/**
+ * External dependencies
+ */
+import $ from 'jquery';
+
 /**
  * Internal dependencies
  */
+import './custom-inputs';
 import './index.scss';
+
+const applicableProductTypes = new Set( [
+	// Simple product
+	'simple',
+	// Variable product
+	'variable',
+	// Product bundle from WooCommerce Product Bundles
+	'bundle',
+] );
```

Comment:
Thanks for the suggestion!

Since the original execution order of the related processing is:
1. [AssetsHandler->add_many](https://github.com/woocommerce/google-listings-and-ads/blob/338300fc25a2b79acccd3b35a5e6f8f51175c609/src/Admin/Admin.php#L74-L75)
2. [AssetsHandler->register](https://github.com/woocommerce/google-listings-and-ads/blob/338300fc25a2b79acccd3b35a5e6f8f51175c609/src/Infrastructure/GoogleListingsAndAdsPlugin.php#L102-L105)
3. [IntegrationInitializer->register](https://github.com/woocommerce/google-listings-and-ads/blob/338300fc25a2b79acccd3b35a5e6f8f51175c609/src/Integration/IntegrationInitializer.php#L43-L46) and [WooCommerceProductBundles->init_product_types](https://github.com/woocommerce/google-listings-and-ads/blob/338300fc25a2b79acccd3b35a5e6f8f51175c609/src/Integration/WooCommerceProductBundles.php#L137-L143)
4. [AssetsHandler->enqueue_many](https://github.com/woocommerce/google-listings-and-ads/blob/338300fc25a2b79acccd3b35a5e6f8f51175c609/src/Admin/Admin.php#L77-L85)

On the first call, the inline script `glaData` is created along with the script file registration. However, the third call has not yet been made, which results in the supported product types not containing the `'bundle'`.

Therefore, I adjusted the execution order as follows in [aaa](https://github.com/woocommerce/google-listings-and-ads/pull/2087/commits/b8ed263fa2750033829ca63e523ad8460969b80b), so that the script registration will be called after the `IntegrationInitializer->register`.

1. [IntegrationInitializer->register](https://github.com/woocommerce/google-listings-and-ads/blob/b8ed263fa2750033829ca63e523ad8460969b80b/src/Integration/IntegrationInitializer.php#L43-L46) and [WooCommerceProductBundles->init_product_types](https://github.com/woocommerce/google-listings-and-ads/blob/b8ed263fa2750033829ca63e523ad8460969b80b/src/Integration/WooCommerceProductBundles.php#L137-L143)
2. [AssetsHandler->register_many](https://github.com/woocommerce/google-listings-and-ads/blob/b8ed263fa2750033829ca63e523ad8460969b80b/src/Admin/Admin.php#L86): Combines the former `AssetsHandler->add_many` and `AssetsHandler->register`
3. [AssetsHandler->enqueue_many](https://github.com/woocommerce/google-listings-and-ads/blob/b8ed263fa2750033829ca63e523ad8460969b80b/src/Admin/Admin.php#L87)

And removed the former calls of `AssetsHandler->add_many` and `AssetsHandler->register`.